rotkehlchen/chain/evm/tokens.py

```diff
@@ -218,21 +256,26 @@ def detect_tokens(
         - BadFunctionCallOutput if a local node is used and the contract for the
           token has no code. That means the chain is not synced
         """
-        with self.db.conn.read_ctx() as cursor:
-            if only_cache is False:
-                self._detect_tokens(addresses=addresses)
+        all_tokens = GlobalDBHandler().get_evm_tokens(
+            chain_id=self.evm_inquirer.chain_id,
+            exceptions=self._get_token_exceptions(),
+            except_protocols=['balancer'],
```

Comment:
Why this exception? I know it's from the code before but do you know why? I am not sure why only balancer? We should figure it out and add a comment here. Or if it's wrong fix it.